src/FormBuilder/Controllers/FormBuilderResourceController.php

```diff
@@ -77,6 +79,9 @@ public function update(WP_REST_Request $request)
             return rest_ensure_response($requiredFieldsError);
         }
 
+        give(UpdateEmailSettingsMeta::class)->__invoke($form);
+        give(UpdateEmailTemplateMeta::class)->__invoke($form);
+
```

Comment:
any reason we can't use the invokable syntax?

`(new UpdateEmailSettingsMeta())($form)`

Reply:
Either method works, but I prefer resolving from the container, which is mock-able and supports dependency injection in the invoke-able class (neither of which is currently being used).

Otherwise, I would have reached for the invokable syntax if I needed to define the constructor arguments manually, which is not the case here.

Reply:
Gotcha, in general I prefer to call actions outside of the container since they're meant to be isolated in nature.